testing/btest/core/tcp/options.zeek

```diff
@@ -38,6 +38,9 @@ event tcp_options(c: connection, is_orig: bool, options: TCP::OptionList)
 				print fmt("    send ts: %s", o$send_timestamp);
 				print fmt("    echo ts: %s", o$echo_timestamp);
 				break;
+			case 27:
+                                print fmt("     TTL Diff: %s", o$ttl_diff);
+                                break;
```

Comment:
I suppose the trace the test is using doesn't contain option 27 because otherwise the baseline would change. Do you have a trace with option 27 that you could add? And then log all the new fields. (Ok to skip traces for the new weirds I think, not worth the effort it seems.)

Some whitespace tweaking needed here too.

Reply:
Unfortunately, the pcap I tested the code with don't have option 27, I can try getting a scapy created pcap for that option and test with it, if we want to test it against that option too.
The only option I was able to check with one of our pcaps was TFO - 34:

1658331756.796497	CDWjhQ1NKSgdvEJwva	109.238.x.x	49380	x.x.x.x	80	tcp_option_TFO_invalid_len	4	F	zeek	TCP
1658331756.796497	C6brlK1euEJ8Z3j7A7	109.238.y.y	7396	y.y.y.y	80	tcp_option_TFO_invalid_len	4	F	zeek	TCP

Reply:
yeah, I think finding/creating a trace with option 27 would be great if you could do that, otherwise we'd be adding a code path here without exercising it.